### PR TITLE
internal/kokoro: print out commit hash for HEAD

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

File: internal/kokoro/test.sh [hunk 1]
```diff
@@ -15,6 +15,8 @@ set -x
 # cd to project dir on Kokoro instance
 cd git/google_api
 
+git rev-parse HEAD
```
Comment (tbpg) [inline internal/kokoro/test.sh:18]: This should be printed by Kokoro. For example

```
Note: checking out '0621fd1d165ef16d7fdb53720d232c894dde22a4'.
```

From https://source.cloud.google.com/results/invocations/0f3145e4-2b9f-447f-bd03-4c21011daf56/log.

But, good with me if you want to print it here.

Comment (broady) [reply internal/kokoro/test.sh:18]: Oh, missed that.

**Comment by broady** [comment]
Okay... there's the weird thing. The build log associated with this PR says:
```
Note: checking out '0621fd1d165ef16d7fdb53720d232c894dde22a4'.
```

which is HEAD@master right now. I guess the presubmit config is messed up?

**Comment by codyoss** [comment]
Now that we know what was going on, I think we can close this?

**Pull request closed without merging**